tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1428,7 +1428,12 @@ def get_value_from_input_value(input_value: str):
         if value[0] == '[' and value[-1] != ']' or value[0] != '[' and value[-1] == ']':
             raise Error("Wrong syntax to specify value. Use --input \"node_name[shape]->value\"")
         if '[' in value.strip(' '):
-            value = value.replace('[', '').replace(']', '').split(' ')
+            value = value.replace('[', '').replace(']', '')
```

Comment:
you also need to update description for `freeze_placeholder_with_value`. We can't deprecate this option now, though, we had a plan in the past:(